Sources/TeslaSwift/TeslaEndpoint.swift

```diff
@@ -105,7 +105,7 @@ extension Endpoint {
         } else {
             switch self {
                 case .oAuth2Authorization, .oAuth2Token, .oAuth2revoke:
-                    return "https://auth.tesla.com"
+                    return "https://auth.tesla.\(country)"
```

Comment:
I would rather create a new set of switch cases than passing the country code like this:
`case .oAuth2AuthorizationCN, .oAuth2TokenCN, .oAuth2revokeCN:`
what do you think?

Reply:
At the same time, It would be noisier but less complex.

Now imagine there would be another edge-case like CN but for another country, we would add another set of switches?

I didn't have time to reverse engineer all the library to find where those switches would be called.